anaSbitReadout.py

```diff
@@ -0,0 +1,376 @@
+            i), "vfat{0} sbitSize".format(i), 8, 0., 8.)
+        vfat_h_delay[i] = r.TH1F("L1A_Sbit_delay{0}".format(
+            i), "vfat{0} L1A delay".format(i), 4096, 0., 4096.)
+        vfat_h_ch[i].SetXTitle("Chann Num")
+        vfat_h_strip[i].SetXTitle("Strip Num")
+        vfat_h_strip[i].SetFillColorAlpha(r.kBlue, 0.35)
+        vfat_h_ch[i].SetFillColorAlpha(r.kBlue, 0.35)
+
+    # initializing eta 1Dhisto
+    ieta_h_strip = ndict()
+    ieta_h_ch = ndict()
+    ieta_h_sbitSize = ndict()
+    ieta_h_delay = ndict()
+
+    for i in range(1, 9):
+        ieta_h_strip[i] = r.TH1F("eta_Strips_vs_hit{0}".format(
+            i), "i#eta = {0} | i#phi (1,2,3)".format(i), 384, 0., 384.)
+        ieta_h_ch[i] = r.TH1F("eta_Chann_vs_hit{0}".format(
+            i), "i#eta = {0} | i#phi (1,2,3)".format(i), 384, 0., 384.)
+        ieta_h_sbitSize[i] = r.TH1F("eta_Sbitsize_vs_hit{0}".format(
+            i), "i#eta = {0} sbitSize".format(i), 8, 0., 8.)
+        ieta_h_delay[i] = r.TH1F("eta_L1A_Sbit_delay{0}".format(
+            i), "i#eta = {0} L1A delay".format(i), 4096, 0., 4096.)
+        ieta_h_strip[i].SetFillColorAlpha(r.kBlue, 0.35)
+        ieta_h_ch[i].SetFillColorAlpha(r.kBlue, 0.35)
+        ieta_h_strip[i].SetXTitle("Strip num")
+        ieta_h_ch[i].SetXTitle("Chan num")
+
+    # initializing 2Dhisto
+    hh_ieta_strip = ndict()
+    hh_ieta_ch = ndict()
+    hh_ieta_strip[0] = r.TH2I(
+        'ieta_strip', 'Strips summary        (i#phi = 1,2,3);strip number;i#eta', 384, 0, 384, 8, 0.5, 8.5)
+    hh_ieta_ch[0] = r.TH2I(
+        'ieta_ch', 'Channels summary        (i#phi = 1,2,3);chan number;i#eta', 384, 0, 384, 8, 0.5, 8.5)
+
+    # loop over all branch names but the first (evnt num)
+    for branch in brName[1:]:
+        print "Unpacking now", branch + \
+            ",", str(rawData[branch].shape).translate(None, "(),"), "entries"
+        # h is a dummy variable to be printed in case of error
+        for h, word in enumerate(rawData[branch]):
+
+            sbitAddr = ((word) & 0x7FF)
+            # INVALID ADDRESS CHECK
+            if sbitAddr >= 1536:
+                continue
+
+            vfatN[0] = (7 - sbitAddr/192 + ((sbitAddr % 192)/64)*8)
+            sbitSize[0] = ((word >> 11) & 0x7)
+            chHitPerCluster[0] = 2*(sbitSize[0]+1)
+            L1Delay[0] = ((word >> 14) & 0xFFF)
+            eta = vfat_to_etaphi[vfatN[0]][0]
+            phi = vfat_to_etaphi[vfatN[0]][1]
+            # SBIT always includes doublet of adjacent channels
+            vfatCH[0] = 2*(sbitAddr % 64)
+            vfatCH[1] = vfatCH[0] + 1
+            strip[0] = vfat_ch_strips[vfatN[0]]['Strip'][vfatCH[0]]
+            strip[1] = vfat_ch_strips[vfatN[0]]['Strip'][vfatCH[1]]
+            # In case of wrong mapping adjacent channels may not be adjacent strips, which is physically inconsistent
+            if(np.abs(strip[0] - strip[1]) > 1):
+                print 'WARNING: not adjacent strips'
+                time.sleep(3)
```

Comment:
Make the `print` statement python3 compatible and remove the `sleep` command.  If you want this to stand out in a log file then colour the message using the [colors class](https://github.com/cms-gem-daq-project/cmsgemos/blob/b44bef4d8cc328b8646474c4cbffced6d9d44197/gempython/utils/gemlogger.py#L4), e.g.:

```python
print("{}{}{}".format(colors.RED,text,colors.ENDC))
```

Reply:
Done